src/components-styled/sidebar-metric/sidebar-metric.tsx

```diff
@@ -14,7 +14,11 @@ interface SidebarMetricProps<T extends { difference: unknown }> {
   scope: DataScope;
   data: T;
   metricName: ValueOf<MetricKeys<T>>;
-  metricProperty: string;
+  /**
+   * Make metric property optional for odd case where we do not show a metric.
+   * Currently only behavior is doing that.
```

Comment:
```suggestion
   * Currently only 'Behavior' is doing that.
```

I misread 'Behavior' as part of the comment, not as reference.